src/MongoBulkDataMigration.ts

```diff
@@ -290,6 +295,22 @@ export default class MongoBulkDataMigration<TSchema extends Document>
       return { ok: status ? 1 : 0 } as any;
     }
 
+    if(this.options.noBackup) {
+      const result = await collection.updateMany({}, this.migrationInfos.rollback());
```

Comment:
@JulienBara-360 This is some spec trick:
- By default `rollback` is a by-doc update
- When `options.noBackup:true`, then `rollback` is a global kind-of hook

Even if `noBackup` is an edge case, I believe a new prop `options.hooks.postRollback: Function` could underline there is a trick.
- [+] The `updateMany()` is not something that scales (not bulk) and we don't want to maintain this query in the core lib.
- [+] Hook options could be reusable,

Wdyt of adding a new option?